.werft/util/shell.ts

```diff
@@ -14,6 +14,9 @@ export const werft = {
         throw err;
     },
     done: (slice) => console.log(`[${slice}|DONE]`),
+    result: (description: string, channel: string, conclusion: "success" | "failure" | string) => {
```

Comment:
Conclusion is something that's specific to the github channels.
In it's proper form this should be:
```
result: (description: string, channel: string, value: string) => {
    exec(`werft log result -d "${description}" -c "${channel}" ${value}`);
},
```